src/app/(views)/managed-courses/[courseId]/modules/[moduleId]/page.tsx

```diff
@@ -14,6 +30,7 @@ export default function Module() {
   const { courseModules, fetchingModules } = useGetModules({ courseId, moduleId })
   const courseModule = courseModules[0]
   const [deletingModule, setDeletingModule] = useState(false)
+  const [visible, setVisible] = useState(false)
```

Comment:
Use a more descriptive name. What is visible? maybe `const [showDeleteModa, setShowDeleteModal]`?